Sources/Purchasing/StoreKit2/ProductsFetcherSK2.swift

```diff
@@ -67,36 +63,11 @@ private extension ProductsFetcherSK2 {
     }
 
     func cache(products: Set<SK2StoreProduct>) async {
-        let storeFrontIdentifier = await self.invalidateOutdatedProductsIfNeeded()
-
-        self.cachedProductsStorefrontIdentifier = storeFrontIdentifier
         self.cachedProductsByIdentifier += products.dictionaryWithKeys {
             $0.productIdentifier
         }
     }
 
-    /// - Returns: current `Storefront` identifier
-    func invalidateOutdatedProductsIfNeeded() async -> String? {
-        let storeFrontIdentifier = await self.currentStorefrontIdentifier
```

Comment:
We don't have to do this validation because now, we manage the Storefront changes and clear the caches.

Reply:
👍🏻